### PR TITLE
Feature build image from branch

### PR DESCRIPTION
**Pull request by walkjivefly**
Adds an extra input field to the CREATE IMAGE FROM TEMPLATE workflow which asks for a branchpath of where to find the manifest.json (yes, needs to be updated in light of yesterday's decision to make manifest-latest.json the ultimate authority). If no path is given it uses blockchain-configuration-files/master

### CODE REVIEW

**Comment by walkjivefly** [comment]
Using it to build PIVX failed; PIVX needs some "sapling" parms at runtime which aren't created by the build from source.
Using it to build SYS failed; there's a missing URI for one of the depends (https://github.com/syscoin/syscoin/issues/465).

But it built DASH v0.17.0.3 correctly and I'll be testing it with more coins over the next few days.

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -14,19 +17,19 @@ on:
...
+    runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
     - name: Login to DockerHub
       run: echo ${{ secrets.docker_password }} | docker login --username ${{ secrets.docker_username }} --password-stdin
 
     # Build images
     - name: INFO BUILD
-      run: echo ${{ github.event.inputs.image }} ${{ github.event.inputs.version }}
+      run: echo ${{ github.event.inputs.branch }} ${{ github.event.inputs.image }} ${{ github.event.inputs.version }}
```
Comment (ConanMishler) [inline .github/workflows/build.yaml:28]: I think this line 28 was meant to use `github.event.inputs.path` instead of `github.event.inputs.branch`, no?

**Review by ConanMishler — COMMENTED** [review]
I'm sure this works fine as is. However, I was thinking it might be nice to give the user an example of how to fill in the `path` field with the URL of the config files. One easy way to do this would be to use the same mechanism that's used in `build.yaml` for filling in the `version` field with the default value, `latest`. (See `build.yaml` lines 12-15.) For the `path` default, we could set the default in the `.yaml` as, 'https://raw.githubusercontent.com/blocknetdx/blockchain-configuration-files/master'. Filling in the *default* config URL like this will make it clear that any alternate URL the user provides needs to be a full path URL that points to the directory where `manifest-latest.json` is stored.

If you go ahead and implement this change (and the minor change of replace `branch` with `path` on line 28), you might as well update `manifest.json` to `manifest-latest.json` while you're in there changing things...

**Comment by atcsecure** [comment]
@walkjivefly can you rebase out those WIP commits?

**Comment by walkjivefly** [comment]
I'll rework it taking into account the changes from PR #41

**Review by ConanMishler — APPROVED** [review]
LGTM. Lots of good attention to detail there! 👍
I looked over you changes pretty thoroughly... much more thoroughly than @shrnkld has time to do, I'm sure.
I seem to have elevated privileges in this repo. It is giving me the option to go ahead and merge your changes, which I will now do.

**Pull request merged**